packages/components/src/components/hds/form/section/multi-field-group/item/index.hbs

```diff
@@ -0,0 +1,10 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Layout::Flex::Item
```

Comment:
[todo] mmm, I see what you've done now, but why? what is the benefit of it, vs just having a `<div>` with a `width` applied? 🤔

Reply:
Because this was based off the `Layout::Flex` component, and I was also thinking we could possibly want to expose some of the other `Item` properties, although I started with making it more simple.

Reply:
Replaced with a div